kobo/apps/openrosa/libs/utils/logger_tools.py

```diff
@@ -153,6 +155,7 @@ def create_instance(
     uuid: str = None,
     date_created_override: datetime = None,
     request: Optional['rest_framework.request.Request'] = None,
+    check_usage_limits: bool = True
```

Comment:
I thought we would need `check_usage_limits=True` at some points in the tests. It seems it is not the case.
So what about only use `if ... and not settings.TESTING`  in `create_instance` directly? 

Do you think it could be useful for future purposes?